<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastros_adicionar_ficha01</class>
 <widget class="QMainWindow" name="cadastros_adicionar_ficha01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>841</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1290</width>
    <height>841</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1290</width>
    <height>841</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Borda">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>783</y>
      <width>1290</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imagens/Borda.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fundo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1251</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Bold</family>
      <pointsize>65</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:7px;
border-radius: 0px;
background-color: rgb(224, 227, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagem_classe">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>531</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/Responsavel_02.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_avancar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;OI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-width:6px;
border-radius: 30px;

font: 75 35pt &quot;Bosch Sans Bold&quot;;
background-color: rgb(0,136,74);</string>
    </property>
    <property name="text">
     <string>AVANÃ‡AR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;
background-color: rgb(193, 194, 204);

font: 75 35pt &quot;Bosch Sans Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>     NOME COMPLETO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_data_nascimento">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;
background-color: rgb(193, 194, 204);

font: 75 35pt &quot;Bosch Sans Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>  DATA DE NASCIMENTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_cancelar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>600</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;OI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-width:6px;
border-radius: 30px;

font: 75 34pt &quot;Bosch Sans Bold&quot;;
background-color: rgb(255,207,0);</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_edv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;
background-color: rgb(193, 194, 204);

font: 75 35pt &quot;Bosch Sans Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>                  EDV</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
